packages/screens/RWA/components/RWAScreenContainer/Header.tsx

```diff
@@ -0,0 +1,181 @@
+          </View>
+        </Pressable>
+        <Pressable style={{ marginRight: 20 }} onPress={toggleSidebar}>
+          <SVG
+            width={32}
+            height={32}
+            source={isSidebarExpanded ? hamburgerCrossSVG : hamburgerSVG}
+            color={theme.textColor}
+          />
+        </Pressable>
+        <ConnectWalletModal
+          visible={isConnectWalletVisible}
+          onClose={toggleConnectWallet}
+        />
+      </View>
+    </View>
+  );
+};
+
+export const Header: React.FC<HeaderProps> = ({ headerTitle, onBackPress }) => {
+  const theme = useTheme();
+  const isLightTheme = useSelector(selectIsLightTheme);
+  const [isConnectWalletVisible, setIsConnectWalletVisible] = useState(false);
+  const selectedWallet = useSelectedWallet();
+
+  return (
+    <View
+      style={{
+        ...HeaderContainerCStyle,
+        backgroundColor: theme.headerBackgroundColor,
+        borderBottomColor: theme.borderColor,
+      }}
+    >
+      <View style={HeaderRowCStyle}>
+        <View style={{ flexDirection: "row", alignItems: "center" }}>
+          <SpacerRow size={8} />
+          {onBackPress && <BackButton onPress={onBackPress} />}
+          <SpacerRow size={1} />
+          <BrandText numberOfLines={1} style={{ fontSize: 20 }}>
+            {headerTitle}
+          </BrandText>
+        </View>
+        {selectedWallet ? (
+          <View style={{ marginRight: 18 }}>
+            <SecondaryButton
+              textStyle={{ fontWeight: "200" }}
+              backgroundColor={theme.tertiaryButtonColor}
+              color={isLightTheme ? theme.secondaryTextColor : theme.textColor}
+              text="My Account"
+              size="SM"
+              width={188}
+            />
+          </View>
+        ) : (
+          <View style={{ flexDirection: "row" }}>
+            <View style={{ marginRight: 18 }}>
+              <SecondaryButton
+                textStyle={{ fontWeight: "200" }}
+                backgroundColor={theme.secondaryButtonColor}
+                color={theme.textColor}
+                text="Register"
+                size="SM"
+                width={188}
+              />
+            </View>
+            <View style={{ marginRight: 18 }}>
+              <SecondaryButton
+                textStyle={{ fontWeight: "200" }}
+                backgroundColor={theme.primaryButtonColor}
+                color={theme.secondaryTextColor}
+                text="Connect Wallet"
+                size="SM"
+                width={188}
+                onPress={() => setIsConnectWalletVisible(true)}
+              />
+              <ConnectWalletModal
+                visible={isConnectWalletVisible}
+                onClose={() => setIsConnectWalletVisible(false)}
+              />
+            </View>
+          </View>
+        )}
+      </View>
+    </View>
+  );
+};
+
+const HeaderContainerCStyle: ViewStyle = {
+  flex: 1,
+  height: headerHeight,
+  maxHeight: headerHeight,
```

Comment:
Please use `MOBILE_HEADER_HEIGHT` in `HeaderMobile` (80px desktop and 64px mobile)